File: modul-04/bahasa c/PRAK405-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

void main()
{
    int n, KELIPATAN;
    int TOTAL = 0;

    printf("Nilai n dan kelipatan : ");
    scanf("%d %d", &n, &KELIPATAN);

    for (int i = 1; i <= n; i++)
    {

        int TOTAL_BARIS = 0;

        for (int j = i; j > 0; j--)
        {

            TOTAL_BARIS += j * KELIPATAN;
            if (j == i)
            {
                printf("(%d * %d)", j, KELIPATAN);
            }
            else
            {
                printf(" + (%d * %d)", j, KELIPATAN);
            }
        }

        printf(" = %d\n", TOTAL_BARIS);
        TOTAL += TOTAL_BARIS;
    }

    printf("%d\n", TOTAL);
}

File: modul-04/python/PRAK405-2310817310003-Mfaridsetiawan.py
n, KELIPATAN = map(int, input("Nilai n dan kelipatan: ").split())
TOTAL = 0

for i in range(1, n + 1):
    TOTAL_BARIS = 0

    for j in range(i, 0, -1):
        TOTAL_BARIS += j * KELIPATAN
        if j == i:
            print(f"({j} * {KELIPATAN})", end="")
        else:
            print(f" + ({j} * {KELIPATAN})", end="")

    print(f" = {TOTAL_BARIS}")
    TOTAL += TOTAL_BARIS

print(TOTAL)